DEVELOPER_GUIDE.md

```diff
@@ -56,6 +53,9 @@ First create a fork of this repo and clone it locally. You should then change to
 ./gradlew clean assemble
 ```
 
+>Worth noting:\
```

Comment:
Please update this section to provide more detail on how to find this file as @cwperks mentioned in his https://github.com/opensearch-project/security/pull/2582#issuecomment-1481144513

Reply:
Note, I would prefer an option that was faster than a full build, such as referencing a part of the build.gradle or using a fast command line option

https://github.com/opensearch-project/security/blob/575c2bc1e7283398ef448c404f480bd7b1eddef5/build.gradle#L18

https://github.com/opensearch-project/security/blob/575c2bc1e7283398ef448c404f480bd7b1eddef5/.github/workflows/ci.yml#L133

Reply:
Thank you @peternied for outlining the easier and faster way to find compatible OpenSearch version!

Probably then it is better to remove the method which includes looking for `plugin-descriptor.properties` file in a build. It is harder and takes more time, as well as it is quite lengthy to describe.